Tools/autotest/arducopter.py

```diff
@@ -7769,39 +7945,257 @@ def AP_Avoidance(self):
         self.context_pop()
 
     def PAUSE_CONTINUE(self):
+
+        # load auto mission
         self.load_mission("copter_mission.txt", strict=False)
 
+        # start mission
         self.set_parameter("AUTO_OPTIONS", 3)
-        self.change_mode('AUTO')
+        self.change_mode("AUTO")
         self.wait_ready_to_arm()
         self.arm_vehicle()
 
-        self.wait_waypoint(4, 4)
-        self.run_cmd(mavutil.mavlink.MAV_CMD_DO_PAUSE_CONTINUE,
-                     0, # param1
-                     0, # param2
-                     0, # param3
-                     0, # param4
-                     0, # param5
-                     0, # param6
-                     0 # param7
-                     )
+        # wait for the waypoint
+        self.wait_current_waypoint(3, 500)
+
+        # send pause command
```

Comment:
Seriously?  *that's* what the next line is doing?!  Huh.

Remove or improve :-)